classes/rnative.html

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: "Introduction to React Native"
+permalink: /classes/rnative
+---
+<h1 class="title page-title">Introduction to React Native</h1>
```

Comment:
I think the titles here should match the title on the homepage, which is currently "App Development with React Native".